rbac/management/cache.py

```diff
@@ -197,17 +197,15 @@ def delete_all_policies_for_tenant(self):
             logger.info("Deleting entire policy cache for tenant %s", self.tenant)
             try:
                 key = self.key_for("*")
-                logger.info("value of key", key)
+                logger.info(f"value of key: {key}")
             except Exception as e:
-                logger.error('error from key = self.key_for("*")', e)
-                raise e
+                logger.error(f'error from key = self.key_for("*") with error message: {e}')
```

Comment:
shouldnot be here `"` instead of ` ?

Reply:
no because we already use `"` for `"*"`
so one option is to use single quotes or we have to escape the inner double quotes
`logger.error(f"error from key = self.key_for(\"*\") with error message: {e}")`